source/developers-guide/plugin-configuration/index.md

```diff
@@ -0,0 +1,685 @@
+---
+layout: default
+title: Plugin configuration
+github_link: developers-guide/plugin-configuration/index.md
+indexed: true
+---
+This document will give you a brief introduction about how to set configuration options for plugins, which parameters are available and
+how to use them.
+
+<div class="toc-list"></div>
+
+## Introduction
+After getting a short introduction by reading the Plugin Quick Introduction, we will now be introduced to
```

Comment:
somewhere here you should mention, that this guide is only valid for the legacy plugin system. with a link to the guide for the new plugin system and its configuration options